### PR TITLE
Migrated runs-on to self hosted

### PR DESCRIPTION
**Pull request by vladciob**
Migrated runners to self hosted in order for fixing the compliance on https://github.com/opentap/KS8400/issues/1068

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,20 +11,27 @@ env:
...
     steps:
       # The get-gitversion action installs OpenTAP and fetches with fetch-depth: 0
       - name: GitVersion
         id: gitversion
-        uses: opentap/get-gitversion@v1.1
+        uses: opentap/get-gitversion@main
 
   Build:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Windows]
```
Comment (alnlarsen) [inline .github/workflows/ci.yml:30]: Can we switch this back to Linux runners?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -63,27 +78,38 @@ jobs:
             bin/Release/*.TapPackage
 
   UnitTests:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Windows]
```
Comment (alnlarsen) [inline .github/workflows/ci.yml:77]: Can we switch this back to Linux runners?

**Review by alnlarsen — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vladciob) [reply .github/workflows/ci.yml]: restored back

File: .github/workflows/ci.yml (hunk 2 above)
Comment (vladciob) [reply .github/workflows/ci.yml]: restored back

**Review by alnlarsen — APPROVED** [review]
lgtm

**Pull request merged**